corehq/apps/reports/standard/inspect.py

```diff
@@ -65,7 +65,11 @@ def _get_users_filter(self, mobile_user_and_group_slugs):
                                        mobile_user_and_group_slugs,
                                        self.request.couch_user)
                     .values_list('_id', flat=True))
-        return form_es.user_id(user_ids)
+        # If no filters are selected, return all results
+        if user_ids:
+            return form_es.user_id(user_ids)
+        else:
+            return 'match_all'
```

Comment:
`match_all` is a [function](https://github.com/dimagi/commcare-hq/blob/2c8d5f343f8342f268f8fb001f4de08ad06cb6b8/corehq/apps/es/filters.py#L21-L22), this will just add a string to the ES query, which I expect would fail.